app/services/tenor.js

```diff
@@ -17,14 +17,18 @@ export default class TenorService extends Service {
     @tracked columns = null;
     @tracked error = null;
     @tracked gifs = new TrackedArray([]);
-    @tracked searchTerm = '';
-    @tracked loadedType = 'trending';
+    @tracked searchTerm = 'excited';
+    @tracked loadedType = 'search';
 
     _columnHeights = [];
     _nextPos = null;
 
     get apiKey() {
-        return this.config.get('tenor.publicReadOnlyApiKey');
+        return this.config.get('tenor.apiKey');
```

Comment:
❔ Why rename the API key in the config system?

Reply:
The property in the new tenor API has been modified. It doesn't represent a Public Read Only Key anymore, but a Google API api-key.